core/src/com/mygdx/game/gameobjects/AlmuBotSimple.kt

```diff
@@ -112,6 +112,25 @@ class AlmuBotSimple(val botId: Int,
         }
     }
 
+    fun reset() {
+        speed = Vector2(0f, 0f)
+        shoot = false
+        life = START_LIFE
+        dead = false
+        ammo = START_AMMO
+        score = 0
+
+        respawnCounter = 0.0
+        cooldownCounter = LITTLE_NEG
+        reloadCounter = LITTLE_NEG
+        previousPosition = Vector2(0f, 0f)
```

Comment:
Wszystkie boty startują z (0,0) ?

Reply:
Boty startują na rand pozycji, pozycja po resecie to ich pozycja przed resetem, previousPosition służy do zarządzania kolizjami, na początek jest (0,0), bo w pierwszym kroku 'nie istnieje' poprzednia pozycja

Reply:
👍